src/devtools/client/inspector/rules/models/element-style.ts

```diff
@@ -399,12 +399,13 @@ export default class ElementStyle {
       // overridden state has changed for the text property.
       // _hasUpdatedCSSVariable will return true if the declaration contains any
       // of the updated CSS variable names.
-      if (
-        this._updatePropertyOverridden(textProp) ||
-        this._hasUpdatedCSSVariable(textProp, changedVariableNamesSet)
-      ) {
-        textProp.updateEditor();
-      }
+      // if (
+      //   this._updatePropertyOverridden(textProp) ||
+      //   this._hasUpdatedCSSVariable(textProp, changedVariableNamesSet)
+      // ) {
+      //   textProp.updateEditor();
+      // }
```

Comment:
What do you think about just deleting this code? Do you think it's very likely that we'll uncomment it soon?

Reply:
You're right, this was just a bad old habit.